### PR TITLE
Liveliness probe does not log a failure - Follow Up

### PR DESCRIPTION
**Pull request by mburke5678**
Adding a comment from BZ after initial merge of https://github.com/openshift/openshift-docs/pull/11916

https://bugzilla.redhat.com/show_bug.cgi?id=1625227#c9

### CODE REVIEW

File: dev_guide/application_health.adoc [hunk 1]
```diff
@@ -104,9 +104,41 @@ livenessProbe:
...
+            - '-c'
+            - timeout 60 /opt/eap/bin/readinessProbe.sh <1>
+        timeoutSeconds: 1
+        periodSeconds: 10
+        successThreshold: 1
+        failureThreshold: 3
+[...] 
+----
+
+<1> Timout value and path to the probe script.
```
Comment (sjenning) [inline dev_guide/application_health.adoc]: type: Timout

File: dev_guide/application_health.adoc [hunk 2]
```diff
@@ -104,9 +104,41 @@ livenessProbe:
...
 ----
 ====
+
 [NOTE]
 ====
-The `timeoutSeconds` parameter has no effect on the readiness and liveness probes for Container Execution Checks.
+The `timeoutSeconds` parameter has no effect on the readiness and liveness
+probes for Container Execution Checks. You can implement a timeout 
+inside the probe itself, as {product-title} cannot timeout on an exec call into
+the container. One way to implement a timeout in a probe is using the `timeout` command to run your
```
Comment (sjenning) [inline dev_guide/application_health.adoc]: s/using the `timeout` command/using `timeout`/  avoids two "command"

File: dev_guide/application_health.adoc [hunk 3]
```diff
@@ -104,9 +104,41 @@ livenessProbe:
...
+        exec:
+          command:
+            - /bin/bash
+            - '-c'
+            - timeout 60 /opt/eap/bin/livenessProbe.sh <1>
+        timeoutSeconds: 1
+        periodSeconds: 10
+        successThreshold: 1
+        failureThreshold: 3
+      readinessProbe:
```
Comment (sjenning) [inline dev_guide/application_health.adoc]: one probe example is enough.  don't need both a liveness *and* readiness probe.

**Comment by mburke5678** [comment]
@mdshuai Please take a look. I added the support work around at Sanket N's request in the BZ.

**Comment by mburke5678** [comment]
@mdshuai BUMP

**Comment by mdshuai** [comment]
Confirmed,  lgtm.

**Comment by mburke5678** [comment]
@openshift/team-documentation Small change. PTAL

**Comment by mburke5678** [comment]
@mdshuai Thanks for the review!

**Review by kalexand-rh — COMMENTED** [review]
two picks and LGTM

**Pull request merged**